client/blocks/reader-join-conversation/dialog.jsx

```diff
@@ -0,0 +1,56 @@
+import { Dialog } from '@automattic/components';
+import { Button } from '@wordpress/components';
```

Comment:
Is there a reason we use the Button from `@wordpress/components` in this file, and use the Button from `@automattic/components` in the example?

Reply:
No reason - its probably just down to how the IDE auto added the import. I'm not sure if it matters but I'll make sure to check when I merge with main PR.